src/applications/disability-benefits/2346/api/index.js

```diff
@@ -0,0 +1,34 @@
+import { apiRequest } from '../../../../platform/utilities/api';
+
+export const getDLCBatteryDataApi = () => apiRequest('/dlc/orderstatus/1014/1');
+// NOTE: Decision was to take socks out of MVP -@maharielrosario at 1/28/2020, 9:49:20 AM
```

Comment:
Should this not be async / awaited since its an API request?

Reply:
```js
async () => await apiRequest('/dlc/orderstatus/1014/1');
```

Reply:
These call have been deleted